.github/workflows/upgrade-python-requirements.yml

```diff
@@ -50,6 +52,6 @@ jobs:
           cd $GITHUB_WORKSPACE/testeng-ci
           python -m jenkins.pull_request_creator --repo-root=$GITHUB_WORKSPACE \
           --target-branch="${{ env.target_branch }}" --base-branch-name="upgrade-python-requirements" \
-          --commit-message="chore: Updating Python Requirements" --pr-title="Python Requirements Update" \
-          --pr-body="Python requirements update.Please review the [changelogs](https://openedx.atlassian.net/wiki/spaces/TE/pages/1001521320/Python+Package+Changelogs) for the upgraded packages." \
-          --user-reviewers="" --team-reviewers="" --delete-old-pull-requests
+          --commit-message="chore: update Python requirements" --pr-title="Python requirements update" \
+          --pr-body="Python requirements update. Please review the [changelogs](https://openedx.atlassian.net/wiki/spaces/TE/pages/1001521320/Python+Package+Changelogs) for the upgraded packages." \
+          --user-reviewers="" --team-reviewers="community-engineering" --delete-old-pull-requests
```

Comment:
So, when we had this previously, we got alerted because the team name didn't work... 🤷‍♀️

Reply:
We had to add the team as an explicit Writer on the repo, which I have now done.  I think this will work.